core/tests/Test.Microsoft.Identity.Core.UIAutomation/UserInformationFieldIds.cs

```diff
@@ -34,7 +34,7 @@ public class UserInformationFieldIds
         public string PasswordInputId { get; set; }
         public string SignInButtonId { get;  set; }
 
-        public void DetermineUser(IUser user)
+        public void DetermineFieldIds(IUser user)
```

Comment:
Agree on the field id's, perhaps UpdateFieldIds ?

Reply:
I feel DetermineFieldIds is more clear as we are determining which ids to use rather than updating them

Reply:
ok, not sure determine is the right word though, but I agree better than update. Perhaps SetFieldIdsFromUser or simply SetFieldIds

---
In reply to: [231373212](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1385#discussion_r231373212) [](ancestors = 231373212)